scalyr_agent/agent_status.py

```diff
@@ -283,6 +285,43 @@ def report_status(output, status, current_time):
     if status.config_status.last_error is not None:
         print >>output, 'Parsing error:         %s' % str(status.config_status.last_error)
 
+    def print_environment():
+
+        # Print scalyr-related env variables in sorted order with critical variables up top. Redact API key.
+        main_keys = ['SCALYR_API_KEY', 'SCALYR_SERVER']
+        special_case_keys = set(['K8S_EVENT_DISABLE'])
+        redacted_keys = set(['SCALYR_API_KEY'])
+
+        # Make a map of uppercase keys -> relevant environment vars (beginning with SCALYR)
+        upper2actualkey = {}
+        for k in os.environ.keys():
+            kup = k.upper()
+            if kup.startswith('SCALYR') or kup in special_case_keys:
+                upper2actualkey[kup] = k
+
+        # Sorted list of all scalyr keys, including main_keys which may not be present
+        # Sort order does not consider letter case.
+        sorted_upperkeys = main_keys + sorted(set(upper2actualkey.keys()) - set(main_keys))
+        # sorted_actualkeys = [upper2actualkey.get(k) or k for k in sorted_upperkeys]
```

Comment:
Looks like some leftover code was left in a comment.

Perhaps delete rather than leaving it in?

Reply:
Thanks for catching it.  Updated.